### DISCUSSION TITLE
Org URL return 404

### DISCUSSION THREAD

**Discussion by jcubic** [zen-fs/core#110] [Q&A] [answered]
the link in the org: https://zen-fs.github.io/ returns 404.

Are there any website for the project? You can get a nice domain at [js.org](https://js.org) for free. it can be easily connected with GitHub pages, all you have to do is to add CNAME file.

**Comment by james-pre** [comment]
The documentation websites are available at https://zen-fs.github.io/<package/repo name>. For example, https://zen-fs.github.io/core has documentation for the core.

I will look at adding a zen-fs.github.io repository to link to the various package documentation.

**Comment by james-pre** [comment, marked as answer]
@jcubic A website has been added at https://zen-fs.github.io which links to the docs
